fastlane/Fastfile

```diff
@@ -106,6 +106,44 @@ end
       release_version: options[:version])
   end
 
+  # This lane lacks documentation because it ought to be part of the localized
+  # metadata download script instead!
+  desc "Updates the files with the localized keywords values for App Store Connect to match the 100 characters requirement"
+  lane :sanitize_appstore_keywords do | options |
+    Dir["./metadata/**"].each do |locale_dir|
+      keywords_path = File.join(locale_dir, 'keywords.txt')
+
+      unless File.exists?(keywords_path)
+        UI.message "Could not find keywords file in #{locale_dir}. Skipping."
+        next
+      end
+
+      keywords = File.read(keywords_path)
+      app_store_connect_keywords_length_limit = 100
+
+      if keywords.length <= app_store_connect_keywords_length_limit
+        UI.verbose "#{keywords_path} has less than #{app_store_connect_keywords_length_limit} characters. Not trimming."
+        next
+      end
+
+      UI.message "#{keywords_path} has more than #{app_store_connect_keywords_length_limit} characters. Trimming it..."
+
+      case File.basename(locale_dir)
+      when 'ar-SA'
+        separator = '،'
+      else
+        separator = ','
+      end
+
+      until keywords.length <= app_store_connect_keywords_length_limit do
+        keywords = keywords.split(separator)[0...-1].join(separator)
+        puts keywords
+      end
+
+      File.write(keywords_path, keywords)
+    end
+  end
+
```

Comment:
Same q as in https://github.com/Automattic/simplenote-ios/pull/1201#discussion_r591356015 here 🤷